plugin/metrics/README.md

```diff
@@ -19,6 +19,7 @@ The following metrics are exported:
 * `coredns_dns_request_type_count_total{server, zone, type}` - counter of queries per zone and type.
 * `coredns_dns_response_size_bytes{server, zone, proto}` - response size in bytes.
 * `coredns_dns_response_rcode_count_total{server, zone, rcode}` - response per zone and rcode.
+* `coredns_dns_programming_latency_seconds{service_kind}` - [DNS programming latency SLI](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/dns_programming_latency.md)
```

Comment:
this metrics should be namespaced in the kubernetes plugin (and moved there) and named
`coredns_kubernetes_programming_latency`

Reply:
Done